yt/python/yt/environment/init_query_tracker_state.py

```diff
@@ -578,6 +578,105 @@ def access_control_objects_mapper(row):
     ),
 ]
 
+
+def get_minus_start_time_mapper(table):
+    column_names = table.user_columns
+
+    def minus_start_time_mapper(row):
+        result = dict([(key, row.get(key)) for key in column_names])
+        result["minus_start_time"] = -row.get("start_time")
+        yield result
+
+    return minus_start_time_mapper
+
+
+def finished_queries_table_by_aco_and_start_time_mapper(row):
+    column_names = FINISHED_QUERIES_TABLE_BY_ACO_AND_START_TIME_V9.user_columns
+
+    result = dict([(key, row.get(key)) for key in column_names])
+    result["minus_start_time"] = -row.get("start_time")
```

Comment:
I meant here we do `start_time -> -(start_time)` and in backend we do `.StartTime -> -i64(.StartTime.Microseconds())`
I hope that `start_time -> -(start_time)` here does the same as backend version. Otherwise we will have values in different formats in the column

Reply:
https://ytsaurus.tech/docs/en/user-guide/storage/data-types#schema_primitive
Internal timestamp representation is "Microseconds"

Additionally, I manually checked that values are in expected format.